03/app.js

```diff
@@ -1,3 +1,10 @@
 export default function randomNumber(min, max) {
-
-}
\ No newline at end of file
+  if (min === 1 && max === 1) return 1;
```

Comment:
Dlaczego tylko 1? Czy to nie jest szczególny przypadek? Przy pisaniu testu jest to ok, ale jako działająca funkcja powinniśmy rozszerzyć to rozwiązanie tj. `if(min === max) return min`. Dodatkowo ten warunek powinien być niżej niż ten z linii 3, aby najpierw sprawdzić poprawność danych :)